modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl

```diff
@@ -0,0 +1,21 @@
+{
+  "Version": "2012-10-17",
+
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "sts:AssumeRole",
+        "s3:PutObject",
+        "s3:ListBucket",
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:iam::${account_number}:role/TDRJenkinsS3ExportRole${stage}",
```

Comment:
Is the `stage` param being converted to title case before being passing into the template?

Reply:
Yeah, it is. I've changed it to `env_title_case` to make it clearer and to match the parameter it comes from.